tests/framework/db/pgsql/CommandTest.php

```
@@ -525,4 +528,145 @@ public function testIssue15827(): void
+                    $found = true;
+                }
+            }
+
+            self::assertTrue(
+                $found,
+                'Unique constraint must exist.',
+            );
+        }
+
+        DbHelper::dropTablesIfExist($db, ['alter_column']);
+    }
```

PHPMD (2.15.0)

[warning] The method testAlterColumn() has a Cyclomatic Complexity of 15. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)